frameworks/keyed/ember/app/templates/components/my-table.hbs

```diff
@@ -1,73 +1,72 @@
-    <div class="row">
-        <div class="col-md-6">
-            <h1>Ember v3.5.1</h1>
+  <div class="row">
+    <div class="col-md-6">
+      <h1>Ember v3.9.1</h1>
+    </div>
+    <div class="col-md-6">
+      <div class="row">
+        <div class="col-sm-6 smallpad">
+          <BsButton id="run" @onClick={{action this.create}}>
+            Create 1,000 rows
+          </BsButton>
+        </div>
+        <div class="col-sm-6 smallpad">
+          <BsButton id="runlots" @onClick={{action this.runLots}}>
+            Create 10,000 rows
+          </BsButton>
+        </div>
+        <div class="col-sm-6 smallpad">
+          <BsButton id="add" @onClick={{action this.add}}>
+            Append 1,000 rows
+          </BsButton>
         </div>
-        <div class="col-md-6">
-            <div class="row">
-                <div class="col-sm-6 smallpad">
-                    {{#bs-button id='run' onClick=(action create)}}
-                      Create 1,000 rows
-                    {{/bs-button}}
-                </div>
-                <div class="col-sm-6 smallpad">
-                    {{#bs-button id='runlots' onClick=(action runLots)}}
-                      Create 10,000 rows
-                    {{/bs-button}}
-                </div>
-                <div class="col-sm-6 smallpad">
-                    {{#bs-button id='add' onClick=(action add)}}
-                      Append 1,000 rows
-                    {{/bs-button}}
-                </div>
-                <div class="col-sm-6 smallpad">
-                    {{#bs-button id='update' onClick=(action update)}}
-                      Update every 10th row
-                    {{/bs-button}}
-                </div>
-                <div class="col-sm-6 smallpad">
-                    {{#bs-button id='clear' onClick=(action clear)}}
-                      Clear
-                    {{/bs-button}}
-                </div>
-                <div class="col-sm-6 smallpad">
-                    {{#bs-button id='swaprows' onClick=(action swapRows)}}
-                      Swap Rows
-                    {{/bs-button}}
-                </div>
-            </div>
+        <div class="col-sm-6 smallpad">
+          <BsButton id="update" @onClick={{action this.update}}>
+            Update every 10th row
+          </BsButton>
         </div>
+        <div class="col-sm-6 smallpad">
+          <BsButton id="clear" @onClick={{action this.clear}}>
+            Clear
+          </BsButton>
+        </div>
+        <div class="col-sm-6 smallpad">
+          <BsButton id="swaprows" @onClick={{action this.swapRows}}>
+            Swap Rows
+          </BsButton>
+        </div>
+      </div>
     </div>
+  </div>
 </div>
 
 
 <table class="table table-hover table-striped test-data">
   <tbody>
-  {{#each this.data key='id' as |item|}}
-
-    <tr class={{if (eq item.id this.selected) 'danger' 'x'}}>
-      <td class="col-md-1">{{item.id}}</td>
+    {{~#each this.data key="id" as |item|~}}
```

Comment:
oh what's this squiggly?

Reply:
it will be replaced (`~`) - remove space

Reply:
Fwiw, the change I did in #506 should have removed *all* unnecessary whitespace, so not sure if that is needed/makes any difference?

Reply:
`~` - removed, thanks!